src/import-data/ImportDataDestination.ts

```diff
@@ -241,7 +241,7 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
 
   const renderSelectTemplateWorkspace = () =>
     h(Fragment, [
-      h2({ style: styles.title }, ['Start with a template']),
+      h2({ style: styles.title }, ['Select a template']),
```

Comment:
Same here, this tense change wasn't explicitly called out in the mockups but I thought it made sense.